terraform/README.md

```diff
@@ -0,0 +1,63 @@
+# Terraform module for kubeflow-volumes
+
+This is a Terraform module facilitating the deployment of the kubeflow-volumes charm, using the [Terraform juju provider](https://github.com/juju/terraform-provider-juju/). For more information, refer to the provider [documentation](https://registry.terraform.io/providers/juju/juju/latest/docs). 
+
+## Compatibility
+This terraform module is compatible with charms of version >= ckf-1.8.
```

Comment:
Should be updated to 1.9, right?

Reply:
I thought 1.8 was compatible, but I checked again and the relations are different, so it makes sense to keep compatibility with 1.9 and up. Thanks!